tests/python/quantization/test_quantization.py

```diff
@@ -214,7 +214,7 @@ def check_quantized_conv(data_shape, kernel, num_filter, pad, stride, no_bias, q
 
 @with_seed()
 def test_quantized_pooling():
-    def check_quantized_pooling(data_shape, kernel, pool_type, pad, stride, global_pool, qdtype):
+    def check_quantized_pooling(data_shape, kernel, pool_type, pad, stride, global_pool, qdtype, convention):
```

Comment:
I suggest we keep this test to test the default behavior without passing in convention. Add another test to pass in convention as an extra argument.

Reply:
Having another check function like `check_quantized_pooling_with_convention` may need much redundant code here. Do you think we can give the argument `convention` a default value here?

Reply:
It's okay to only keep this test method. But As @yuxihu mentioned, you may want to test the backward compatibility of the operator without passing this argument.

Reply:
Sure.